packages/reports/src/Containers/open-positions.jsx

```diff
@@ -400,6 +467,40 @@ const OpenPositions = ({
         setContractTypeValue(e.target.value);
     };
 
+    const getOpenPositionsTable = () => {
+        if (is_options_selected)
+            return (
+                <OpenPositionsTable
+                    is_empty={active_positions_filtered.length === 0}
+                    className='open-positions'
+                    columns={columns}
+                    {...shared_props}
+                    row_size={isMobile() ? 5 : 63}
+                />
+            );
+        if (is_turbos_selected)
+            return (
+                <OpenPositionsTable
+                    className={classNames('open-positions', {
+                        'open-positions-turbos': is_mobile,
+                    })}
+                    columns={columns}
+                    is_empty={active_positions_filtered.length === 0}
+                    row_size={isMobile() ? 5 : 63}
+                    {...shared_props}
+                />
+            );
+        return (
+            <OpenPositionsTable
+                className='open-positions-multiplier open-positions'
+                columns={columns}
+                row_size={isMobile() ? 3 : 68}
+                is_empty={active_positions_filtered.length === 0}
+                {...shared_props}
+            />
+        );
+    };
```

Comment:
```suggestion
        let classname = 'open-positions';
        let row_size = isMobile() ? 5 : 63;
        if (is_turbos_selected) {
            classname = classNames('open-positions', {
                        'open-positions-turbos': is_mobile,
                    });
        } else if (is_multiplier_selected) {
            className = 'open-positions-multiplier open-positions';
            row_size= isMobile() ? 3 : 68;
        }
        return (
              <OpenPositionsTable
                  className={classname}
                  columns={columns}
                  is_empty={active_positions_filtered.length === 0}
                  row_size={row_size}
                  {...shared_props}
              />
        );
    };
```
@hamza-deriv what do you think about this one?🙂

Reply:
done

Reply:
done